### PR TITLE
fix(sessions) add security warning

### PR DESCRIPTION
**Pull request by Tieske**

### CODE REVIEW

**Comment by Tieske** [comment]
ping @kinman fyi

File: app/enterprise/2.1.x/developer-portal/configuration/authentication/sessions.md [hunk 1]
```diff
@@ -60,6 +60,11 @@ The Session configuration is secure by default, so the cookie uses the [Secure,
 * If using HTTP instead of HTTPS: `"cookie_secure": false`
 * If using different subdomains for the [portal_api_url](/enterprise/{{page.kong_version}}/developer-portal/networking/#portal_api_url) and [portal_gui_host](/enterprise/{{page.kong_version}}/developer-portal/networking/#portal_gui_host), see the example below for [Domains](https://docs.konghq.com/enterprise/2.1.x/developer-portal/configuration/authentication/sessions/#domains). 
 
+⚠️**Important:** Sessions are not invalidated when a user logs out if `"storage": "cookie"`
```
Comment (tlblessing) [inline app/enterprise/2.1.x/developer-portal/configuration/authentication/sessions.md:63]: Suggest using a dev format with coloring to make this stand out more.

  <div class="alert alert-warning">
    <i class="fas fa-exclamation-triangle" style="color:orange; margin-right:3px"></i>
    <strong>Important: </strong> Sessions are not invalidated when a user logs out if `"storage": "cookie"`
    (the default) is used. In that case, the cookie is deleted client-side. Only when session data is 
    stored server-side with `"storage": "kong"` set is the session actively invalidated.
    </div>

File: app/enterprise/2.1.x/kong-manager/authentication/sessions.md [hunk 2]
```diff
@@ -70,6 +70,11 @@ directives.
 * If using HTTP instead of HTTPS: `"cookie_secure": false`
 * If using different domains for the Admin API and Kong Manager: `"cookie_samesite": "off"`
 
+⚠️**Important:** Sessions are not invalidated when a user logs out if `"storage": "cookie"`
+(the default) is used. In that case the cookie is deleted client-side. Only when session data is
```
Comment (tlblessing) [inline app/enterprise/2.1.x/kong-manager/authentication/sessions.md:74]: Same comment as above. At least note the punctuation corrections:

```suggestion
(the default) is used. In that case, the cookie is deleted client-side. Only when session data is
```

File: app/enterprise/2.1.x/kong-manager/authentication/sessions.md [hunk 3]
```diff
@@ -70,6 +70,11 @@ directives.
...
+stored server-side with `"storage": "kong"` set, the session is actively invalidated.
```
Comment (tlblessing) [inline app/enterprise/2.1.x/kong-manager/authentication/sessions.md:75]:
```suggestion
stored server-side with `"storage": "kong"` set is the session actively invalidated.
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Suggested edits.

**Review by tlblessing — APPROVED** [review]
@Tieske thank you for the PR!

**Pull request merged**